pages/_app.js

```diff
@@ -3,18 +3,25 @@
  * Wraps every page around a theme provider
  */
 
-import App from 'next/app'
-import React from 'react'
+import App from 'next/app';
+import React from 'react';
+import withRedux from 'next-redux-wrapper';
 import { ThemeProvider } from 'styled-components';
-import theme from "../components/theme";
+import { Provider } from 'react-redux';
+import store from '../data/store';
+import theme from '../components/theme';
 
-export default class MyApp extends App {
-  render () {
-    const { Component, pageProps } = this.props
-    return (
-      <ThemeProvider theme={theme}>
-        <Component {...pageProps} />
-      </ThemeProvider>
-    )
-  }
-}
\ No newline at end of file
+class MyApp extends App {
+    render() {
+        const { Component, pageProps } = this.props;
+        return (
+            <ThemeProvider theme={theme}>
```

Comment:
Should we wrap Provider around ThemeProvider instead? That way we can temporarily change themes and manage that with redux.

Reply:
Make sense. Thanks, I made the changes